### DISCUSSION TITLE
Read in audio mode

### DISCUSSION THREAD

**Discussion by Zorglub44** [sumatrapdfreader/sumatrapdf#3603] [General]
hello everyone

I would like to know if it is possible to read in audio mode a pdf file in sumatra. If possible, please tell me how to activate the reading mode because I can’t find anything in the documentation.

Thank's

**Comment by GitHubRulesOK** [comment]
The "reading" ability of a document depends on the extraction of plain text content so not possible in some formats that SumatraPDF "reads" and for many the plain text is not in audio order so for example when you use "ReadAloud" in Chrome / Edge the words can jump all over the page in nonsensical fashion or sound like gibberish since the page is just characters / letter / glyphs and symbols. 

What a larger application like Acrobat can do (just for PDF) is analyse the page order of the extracted characters and produce a more logical humanist extraction then offer that to an audio function. you may see parts of those text extractor modules, as separate programs they even have one for non PDF content !
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6f6f4e16-b3d4-418e-bc52-5a1f36e14d0e)

SumatraPDF as a text viewer is limited in what Nrrator can use from the screen and there have been varios mixed results over the years thus the simplest way to read a page is pass the page over to an external dedicated TTS app.

A very good app to read many file types is Balabolka  http://www.cross-plus-a.com/balabolka.htm

>"Balabolka is one of the most efficient, free text-to-speech software,"  
Balabolka supports text file formats: AZW, AZW3, CHM, DjVu, DOC, DOCX, EML, EPUB, FB2, FB3, HTML, LIT, MD, MOBI, ODP, ODS, ODT, PDB, PRC, PDF, PPT, PPTX, RTF, TCR, WPD, XLS, XLSX.  

and it can read a given word, line, passage or page(s) if sent from SumatraPDF to one of its many variant applications (several methods).

**Reply by Zorglub44** [reply]
Thank's
